lib/screens/home/widgets/flower/petal_icon_button.dart

```diff
@@ -16,6 +16,7 @@ class PetalIconButton extends StatelessWidget {
           shape: BoxShape.circle,
         ),
         child: CustomIconButton(
+          highlightColor: Colors.black,
```

Comment:
What is this used for?

Reply:
I think it's very likely a leftover from when I was testing how different params affect the buttons and I just forgot to remove it. I'll make sure it's fine to delete it a bit later today.

If that's all the comments you have for this, can I merge this branch after this fix/check?

Reply:
Oh it's the color of when you are pressing the button. It's just quite small so it's prolly mostly covered by your finger. Still, it's a leftover from my experiements so I'll prolly delete it.

Reply:
Okay, I was just wondering if we want to leave it like that or not.